### PR TITLE
csview 1.0.1 new formula

### PR DESCRIPTION
**Pull request by wfxr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

[csview](https://github.com/wfxr/csview) is a high performance csv viewer for cli with emoji and [CJK](https://en.wikipedia.org/wiki/CJK_characters) character support.
screenshot

### CODE REVIEW

File: Formula/csview.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    system "cargo", "install", *std_cargo_args
+
+    zsh_completion.install  "completions/zsh/_csview"
+    bash_completion.install "completions/bash/csview.bash"
+    fish_completion.install "completions/fish/csview.fish"
+  end
+
+  test do
+    (testpath/"test.csv").write("a,b,c\n1,2,3")
+    assert_equal "┌───┬───┐\n│ a │ c │\n├───┼───┤\n│ 1 │ 2 │\n└───┴───┘\n", shell_output("#{bin}/csview #{testpath}/test.csv")
```
Comment (carlocab) [inline Formula/csview.rb:20]:
```suggestion
    assert_equal <<~EOS, shell_output("#{bin}/csview #{testpath}/test.csv")
      ┌───┬───┐
      │ a │ c │
      ├───┼───┤
      │ 1 │ 2 │
      └───┴───┘
    EOS
```

Comment (wfxr) [reply Formula/csview.rb:20]: Thank you!

**Review by carlocab — APPROVED** [review]
Thanks!

**Review by SMillerDev — APPROVED** [review]
Thanks @wfxr  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**